strawberryfields/backends/fockbackend/ops.py

```diff
@@ -500,3 +498,68 @@ def hermiteVals(q_mag, num_bins, m_omega_over_hbar, trunc):
         Hvals[i] = 2 * x * Hvals[i - 1] - 2 * (i - 1) * Hvals[i - 2]
 
     return q_tensor, Hvals
+
+
+def gkp_displacements(t, k, epsilon):
+    """
+    Helper function to generate the displacements parameters associated with the teeth of
+    GKP computational basis state k.
+    """
```

Comment:
If looking good, worth checking the types and description before accepting.
```suggestion
    """Helper function to generate the displacements parameters associated with the teeth of
    GKP computational basis state k.
    
    Args:
        t (array): the teeth of GKP computational basis
        k (int): a computational basis state label, can be either 0 or 1
        epsilon (float): finite energy parameter of the state

    Returns:
        array: the displacements
```